### PR TITLE
Remove support for py36 and py37

### PR DESCRIPTION
**Pull request by bonddim**
Depends-on: https://github.com/ansible-community/molecule/pull/3436

### CODE REVIEW

File: .github/workflows/tox.yml [hunk 1]
```diff
@@ -22,12 +22,6 @@ jobs:
...
           - tox_env: lint
           # - tox_env: docs
-          - tox_env: py36
-            PREFIX: PYTEST_REQPASS=2
-          - tox_env: py36-devel
-            PREFIX: PYTEST_REQPASS=2
-          - tox_env: py37
-            PREFIX: PYTEST_REQPASS=2
           - tox_env: py38
             PREFIX: PYTEST_REQPASS=2
```
Comment (ziegenberg) [inline .github/workflows/tox.yml:26]: might be a good idea to move the first `devel` target and create an additional `py38-devel`.
```suggestion
          - tox_env: py38
            PREFIX: PYTEST_REQPASS=2
          - tox_env: py38-devel
            PREFIX: PYTEST_REQPASS=2
```

File: tox.ini [hunk 2]
```diff
@@ -4,8 +4,8 @@ minversion = 3.18.0
 envlist =
     lint
     packaging
-    py{36,37,38,39,310}
-    py{36,37,38,39,310}-{devel}
+    py{38,39,310}
```
Comment (ziegenberg) [inline tox.ini:7]: I'm afraid you missed lines 60 and 61 in this file. They also reference `py36` and `py37`.

**Review by ziegenberg — COMMENTED** [review]
Looks good to me. I'll start the GHA on this PR.

**Review by ssbarnea — APPROVED** [review]

File: tox.ini (hunk 2 above)
Comment (bonddim) [reply tox.ini:7]: Thanks!
Will review and fix on next PR

**Pull request merged**